doc/src/cse-user-manual.yaml

```diff
@@ -28,13 +28,32 @@ html-site-top: "/index.html"
 html-site-toc: "/cse-user-manual/index.html"
 css-files:
   - "/css/base.css"
+ignore-coverage:
+  records:
+    - "airHandler"
+    - "boiler"
+    - "chiller"
+    - "coolPlant"
+    - "heatPlant"
+    - "inverse"
+    - "terminal"
+    - "towerPlant"
+<% if inactive_CNE_records %>
+  data-fields:
+    export:
+      - "exTu"
+<% end %>
 sections:
   - [1, "introduction.md"]
   - [1, "operation.md"]
   - [1, "input-structure.md"]
   - [1, "input-data.md"]
   - [2, "records/top-members.md"]
   - [2, "records/holiday.md"]
+  - [2, "records/descond.md"]
+<% if inactive_CNE_records %>
+  - [2, "records/inverse.md"]
+<% end %>
```

Comment:
This isn't needed. I don't think we'll ever need this in the documentation. It sounds like it was a half-developed feature that won't be used. Right, @chipbarnaby?